queue_job/models/queue_job.py

```diff
@@ -127,38 +128,49 @@ class QueueJob(models.Model):
     worker_pid = fields.Integer(readonly=True)
 
     def init(self):
+        # Odoo 19: self._cr deprecated; use self.env.cr.
+        # Prefer tools.sql helpers for idempotent DDL.
```

Comment:
These Odoo 19 comments will not help much we we look at future versions. They were helpful for code review, but now it's better to remove them.